packages/plugin-dom-editable/src/DomEditable.ts

```diff
@@ -107,8 +112,14 @@ export class DomEditable<T extends JWPluginConfig = JWPluginConfig> extends JWPl
                     }
                     return [
                         'deleteForward',
-                        {},
-                        VRange.selecting(this.editor.selection.range.start, end),
+                        {
+                            context: {
+                                range: new VRange(
+                                    this.editor,
+                                    VRange.selecting(this.editor.selection.range.start, end),
+                                ),
+                            },
+                        },
```

Comment:
Non ! Là tu viens de créer un range qui ne sera jamais supprimé !

Reply:
J'ai donné une alternative ou on supprime le range a la main apres l'avoir utilisé.
```ts
const [commandName, commandParams] = commandSpec;
if (commandName) {
    await execCommand(commandName, commandParams);
    if (commandParams.range) range.remove()
}
```
Qu'en pense tu?

Reply:
Mais il y aura toujours `commandParams.context.range`, c'est le range par défaut de l'éditeur.

Reply:
Non parce que dans ce cas ci `commandParams` est l'objet renvoyé par `_matchCommand`. Le params dispatché est créer comme ca: `const context = { ...this.defaultContext, ...paramsContext };`.
Donc le range qu'on supprime est celui retourné par `_matchCommand`.

Reply:
Là t'es censé faire un `VRange.temporary` donc, parce que sinon t'as toujours le même probleme ici. T'as oublié la fonction static en fait. Faut pas faire `new VRange` parce que c'est trop facile de se tromper et d'oublier de le rendre temporaire.

Reply:
What? J'ai pas compris alors. Il me semblait qu'on avait dit qu'on settait temporary sur le constructor.
Tu veux que je fasse :
```js
temporary(arg1, arg2, ar3): VRange {
  return new VRange(arg1, arg2, arg3, true);
}
```
?